extensions/PhabBugz/lib/Feed.pm

```diff
@@ -688,7 +688,7 @@ sub process_new_user {
         # that are connected to revisions
         f11 => 'attachments.filename',
         o11 => 'regexp',
-        v11 => '^phabricator-D[[:digit:]]+-url[[.period.]]txt$',
+        v11 => '^phabricator-D[[:digit:]]+-url.txt$',
```

Comment:
[[.period.]] is a mysql-ism that won't work with sqlite. Technically the regex is more open now, but I think it won't hurt.

Reply:
works for me